content/applications/hr/referrals.rst

```diff
@@ -0,0 +1,707 @@
+
+Points
+~~~~~~
+
+The points that can be earned for a referral are the same across all job positions. Each stage of
+the recruitment process has corresponding points assigned to it. The stages listed correspond to the
+stages configured in the *Recruitment* application.
+
+On each individual referral card, beneath the points earned is a bar chart that displays how many
+points have been earned out of the total possible points that can be earned if the applicant is
+hired.
+
+Beneath the bar chart is a list of the various recruitment stages and the points earned when the
+referral moves to that stage. If a stage has been achieved and the points have been earned, a green
+check mark appears next to the stage.
+
+.. example::
+   The pre-configured point structure for referrals is as follows:
+
+   - :guilabel:`Initial Qualification`: 1 point
+   - :guilabel:`First Interview`: 20 points
+   - :guilabel:`Second Interview`: 9 points
+   - :guilabel:`Contract Proposal`: 5 points
+   - :guilabel:`Contract Signed`: 50 points
+
+   The total points earned for a referral that is hired is 85 points. Modifications of the points
+   earned for each stage is done in the Recruitment application. Refer to the :doc:`../recruitment`
+   documentation to modify the points for each stage.
+
+.. _referrals/rewards:
+
+Rewards
+=======
+
+Create rewards
+--------------
+
+The rewards are the only configurations needed when setting up the Referrals application.
+To add rewards, navigate to :menuselection:`Referrals app --> Configuration --> Rewards`. Click
+:guilabel:`Create` and a reward form loads. Enter the following information on the form:
+
+- :guilabel:`Product Name`: Enter the name as it should appear for the reward.
+- :guilabel:`Cost`: Enter the amount of points required to redeem the reward.
+- :guilabel:`Company`: Using the drop-down menu, select the company the reward will appear for. If a
+  reward is to be used for several companies, each company needs to have a record of the reward,
+  with the company listed on the form.
+- :guilabel:`Gift Responsible`: Select the person who will be responsible for procuring and
+  delivering the reward to the recipient, using the drop-down menu. This person is alerted when the
+  reward is purchased, so they know when to deliver the reward to the recipient.
+- :guilabel:`Description`: Type in the description for the reward. This will be visible on the
+  reward card, beneath the title.
+- :guilabel:`Photo`: Add a photo of the reward, which appears on the rewards page. Hover over the
+  image square, and two icons appear. Click the :guilabel:`✏️ (pencil)` icon and a file explorer
+  appears. Navigate to the photo file, then click :guilabel:`Open` to select it.
+
+.. image:: referrals/rewards.png
+   :align: center
+   :alt: A filled out reward form with all details entered.
+
+The only required fields are the :guilabel:`Product Name`, :guilabel:`Company`, and
+:guilabel:`Description`.
```

Comment:
The only reason I don't agree is because the definitions are immediately above them, and they are too long to add to this chunk-  think it would look repetitive if added.  I did move it up above the image so there is no 'gap' between the listed info and this sentence.